scripts/leetcode_directory_md.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+from urllib.request import Request, urlopen
+from pathlib import Path
+from os import listdir
+import json
+from dataclasses import dataclass

```

Comment:
```suggestion
from dataclasses import dataclass
from os import listdir
from pathlib import Path

import requests
```

It is recommended to sort your imports - I recommend you to install auto formatting tools such as `black` and `flake8` to ensure that you have consistent formatting throughout your code.